.gitignore

```diff
@@ -10,6 +10,7 @@
 *.o
 *.os
 *.so
+*.dll
```

Comment:
Why did you add this, I don't think Nuitka does this, or is about the Python DLLs ? In that case better make it, "python??.dll" which is something actually missing from it. For "installed" Python it should happen, but you probably don't use that?

Reply:
I think when I added that was because when I run some tests few days ago some python??.dll was being added somewhere in the tests, not providing a clean working copy, don't recall which test though